dev/bin/interaction/create-processor-v2.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/sh
+
+usage() {
+    echo "Usage: $0 [OPTIONS]"
+    echo
+    echo 'Creates a V2 processor containing Camel K flows'
+    echo
+    echo 'Options:'
+    echo '  -h    Print this help'
+    echo '  -b    Destination bridge ID (must already exist)'
+    echo '  -n    Name of the processor to be created'
+    echo '  -f    YAML file containing the flows defined as Camel DSL'
+    echo
+    echo 'Environment variables:'
+    echo '  MANAGER_URL    API base URL (default = http://localhost:8080)'
+    echo '  OB_TOKEN       Authentication token'
+}
+
+usage_and_exit() {
+  usage; exit $1;
+}
+
+for tool in curl yq; do
+  which "$tool" &> /dev/null
+  if [ $? -ne 0 ]; then
+    echo "Required tool \"$tool\" is missing"
+    exit 1
+  fi
+done
+
+while getopts "b:f:n:h" i
+do
+    case "$i"
+    in
+        h) usage_and_exit 0 ;;
+        b) bridge_id="${OPTARG}" ;;
+        f) flow_file="${OPTARG}" ;;
+        n) processor_name="${OPTARG}" ;;
+        :) usage_and_exit 1 ;; # If expected argument omitted:
+        *) usage_and_exit 1 ;; # If unknown (any other) option
+    esac
+done
+shift "$((OPTIND-1))"
+
+if [ ! "${bridge_id}" ]; then
+  echo "$0: destination bridge ID is empty"; usage_and_exit 1;
+elif [ ! "${processor_name}" ]; then
+  echo "$0: processor name ID is empty"; usage_and_exit 1;
+elif [ ! "${flow_file}" ]; then
+  echo "$0: flow file is empty"; usage_and_exit 1;
+elif [ ! -f "${flow_file}" ]; then
+  echo "$0: flow file \"${flow_file}\" doesn't exist"; usage_and_exit 1;
+fi
+
+json_flow=$( yq -o=json "$flow_file" )
```

Comment:
If the file is malformed the script should stop, I tried with a not valid yaml and I got

```
 Error: bad file 'README.MD': yaml: line 20: did not find expected <document start>
 + curl -vvv http://localhost:8080/api/smartevents_mgmt/v2/bridges/test/processors -H 'Authorization: bearer ' -H 'Accept: application/json' -H 'Content-Type: application/json' --data-raw '{"name":"test","flows":"Start by making a copy of the `environment-example` file called `environment` and modify it accordingly, it won'\''t be pushed into the repository.\nThen it'\''s important to always have the bearer token set\n```shell\nsource get-token.sh\n```\nAnd then call the others such as\n```shell\n./get-all-bridges.sh ```"'
 *   Trying 127.0.0.1:8080...
 * connect to 127.0.0.1 port 8080 failed: Connection refused
 *   Trying ::1:8080...
 * connect to ::1 port 8080 failed: Connection refused
 * Failed to connect to localhost port 8080 after 10 ms: Connection refused
 * Closing connection 0
 curl: (7) Failed to connect to localhost port 8080 after 10 ms: Connection refused
```

Reply:
added a check for parse failures that blocks the script before executing the API call